### DISCUSSION TITLE
Problem with TMUX

### DISCUSSION THREAD

**Discussion by codeskin-r** [Matt-FTW/dotfiles#32] [Q&A]
![captura-2024-11-12-1731425473](https://github.com/user-attachments/assets/0bab52de-e21e-4791-b714-eb8157fa8486)
Hello! I have problems with TMUX, when I use "prefix + I" it does not install the plugins, and a green strip remains like the screenshot I sent, what am I doing wrong?

**Comment by Matt-FTW** [comment]
Hi,

Sorry to answer a little bit late. So from my testing it seems like its a problem with the tmux plugin manager (tmp). If you see the recent issues, there are some people that cannot install the plugins. I broke my installation trying to test this haha..

Once I have some news, I'll reach out.

**Reply by codeskin-r** [reply]
Dont worry, thanks for the answer!

**Comment by Matt-FTW** [comment]
So it seems that I was wrong about the tmp issues. At least they don't affect the dotfiles setup. 

Did u install the TPM plugin manager? If not, just follow the step on the docs page, and once you have that, press the prefix (which is alt+space by default) and i in caps. You should be able to install them that way.

**Reply by codeskin-r** [reply]
> So it seems that I was wrong about the tmp issues. At least they don't affect the dotfiles setup.
> 
> Did u install the TPM plugin manager? If not, just follow the step on the docs page, and once you have that, press the prefix (which is alt+space by default) and i in caps. You should be able to install them that way.

Hello! Sorry for answering so late, I really appreciate your help, now I'm going to review this, I already assumed I couldn't solve it, now I see it and I'll tell you!

**Reply by codeskin-r** [reply]
> > So it seems that I was wrong about the tmp issues. At least they don't affect the dotfiles setup.
> > Did u install the TPM plugin manager? If not, just follow the step on the docs page, and once you have that, press the prefix (which is alt+space by default) and i in caps. You should be able to install them that way.
> 
> Hello! Sorry for answering so late, I really appreciate your help, now I'm going to review this, I already assumed I couldn't solve it, now I see it and I'll tell you!

I couldn't solve it, the green stripe continues to appear as seen in the screenshot.

**Reply by Matt-FTW** [reply]
Ummm thats really weird, sorry if this sounds rude but are you using the keybinding for installing plugins with TMP? The Alt+Space + I. It's the only thing that I saw that could cause the issue.
